src/components/Books/Book.js

```diff
@@ -3,6 +3,11 @@ import PropTypes from 'prop-types';
 import styles from './books.module.css';
 
 class Book extends PureComponent {
+  remove = () => {
+    const { book, removeBook } = this.props;
+    removeBook(book);
+  };
+
```

Comment:
- [ ] The Prop for remove is not handling the event properly even with the validation, please fix it